README.md

```diff
@@ -100,6 +105,106 @@ check inside pallet revive's [`Cargo.toml`](https://github.com/paritytech/revive
 in order to see which commit of the polkadot-sdk you should use to build the
 `substrate-node` and `eth-rpc` binaries.
 
+## Matter Labs Test Options
+
+### Filtering Contracts
+
+When running matter labs tests with the `--matter-labs` option you can set the `TEST_FILTER` option in order to compile and run only a subset of the contracts.
+
+Example:
+```bash
+./init.sh --ethereum -- --matter-labs -- -- -- -- /destructuring 
```

Comment:
This won't work because of the missing argument.

Reply:
The example should work now